.github/CODEOWNERS

```diff
@@ -1317,6 +1317,7 @@ packages/kbn-monaco/src/esql @elastic/kibana-esql
 /x-pack/solutions/observability/plugins/infra/server/utils @elastic/obs-ux-infra_services-team
 /x-pack/test_serverless/functional/test_suites/observability/infra @elastic/obs-ux-infra_services-team
 /x-pack/test/api_integration/services/infraops_source_configuration.ts @elastic/obs-ux-infra_services-team @elastic/obs-ux-logs-team # Assigned per https://github.com/elastic/kibana/pull/34916
+/x-pack/solutions/observability/plugins/observability/public/pages/overview @elastic/obs-ux-infra_services-team # Assigned to this team since it mostly uses infra/APM components
```

Comment:
How does it work with the more general rule? Does it take the more granular first, or does the order matter? (I always keep forgetting!)

```
x-pack/solutions/observability/plugins/observability @elastic/obs-ux-management-team
```

Reply:
Order matters, last one in the file will be used.